### PR TITLE
Updated assert based on BZ#1778055

### PR DESCRIPTION
**Pull request by vijay8451**
- Updated assert based on BZ:1778055
- Add `upgrade` decorator.
- Test results :
```
PASSED2019-12-11 10:15:56 - robottelo - DEBUG - Finished Test: RepositoryTestCase/test_positive_srpm_list_end_to_end
```

### CODE REVIEW

File: tests/foreman/cli/test_repository.py [hunk 1]
```diff
@@ -1937,7 +1937,7 @@ def test_positive_srpm_list_end_to_end(self):
         # Remove uploaded SRPM
         Repository.remove_content({
             'id': new_repo['id'],
-            'ids': [Srpm.list()[0]['id']],
+            'ids': [Srpm.list({'repository-id': new_repo['id']})[0]['id']],
```
Comment (omkarkhatavkar) [inline tests/foreman/cli/test_repository.py]: what is the difference between id and id's? is that both required while removing the content?

If I'm trying to understand here if I removed the repository it does not deletes the all associated Srpms, I could be wrong?

Comment (vijay8451) [reply tests/foreman/cli/test_repository.py]: Ids => We could pass multiple SRPM packages IDs under it .
id=> id filtering the first package ID only in that repo.

Comment (omkarkhatavkar) [reply tests/foreman/cli/test_repository.py]: means we can have the privilege to delete specific srpms from the uploaded repo?

Comment (vijay8451) [reply tests/foreman/cli/test_repository.py]: yes, this is how it works for every content i.r. rpm/file/ ..etc

Comment (omkarkhatavkar) [reply tests/foreman/cli/test_repository.py]: not for module streams :)

Comment (vijay8451) [reply tests/foreman/cli/test_repository.py]: I mean content which can upload same delete as well :) ... I am not sure if module streams can upload manual :)

**Pull request merged**